src/libraries/Microsoft.PowerFx.Connectors/Execution/OpenApiJsonSerializer.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.PowerFx.Connectors.Execution
+{
+    internal class OpenApiJsonSerializer
+    {
+        private readonly Dictionary<string, object> _dic;
```

Comment:
> private readonly Dictionary<string, object> _dic; [](http://example.com/codeflow?start=7&length=50)

Pass as a parameter instead of as a field. This will be necessary for nested objects.